### ISSUE TITLE
Worker Threads

### ISSUE THREAD

**Issue by svozza** [caolan/highland#674]
It is absolutely trivial to use thew worker threads feature in Node with Highland. I'm not sure if this should be part of core but at the very least I think we should document it:

Example here: https://github.com/svozza/highland-worker-thread

**Comment by vqvu** [comment]
Worker threads are still experimental, so I don't think they belong as part of core yet. But I agree that they should be documented on the website. Probably part of the [Examples section](http://highlandjs.org/#examples).

Do you want to put together the PR for that?

Side note, I think there's a small bug in your code. It should also push `nil` on `error`.

**Comment by svozza** [comment]
Yep, I'll put together a PR today. Good spot on the error, I'm a little rusty. 😉

**Comment by taybin** [comment]
Did this ever happen?
